cypress/integration/pages/mostWatchedPage/tests.js

```diff
@@ -0,0 +1,87 @@
+import path from 'ramda/src/path';
+import config from '../../../support/config/services';
+
+export default ({ service, pageType, variant }) => {
+  describe(`Tests for ${service} ${pageType}`, () => {
+    describe('Most Watched component', () => {
+      before(() => {
+        cy.getToggles(config[service].name);
+      });
+      // Allows constructing most watched path with variant
+      const serviceVariant = variant === 'default' ? '' : `/${variant}`;
+
+      const mostWatchedPath = `/${config[service].name}/mostwatched${serviceVariant}.json`;
+      it('should show/not show the Most Watched component if the toggle is enabled for the service', function test() {
+        // Find if the service has component enabled in toggles
+        cy.fixture(`toggles/${config[service].name}.json`).then(toggles => {
```

Comment:
I thought the method to get toggles was like:

`cy.get('@toggles').then(toggles => {`

Reply:
We needed to make some changes because the memoization meant that services with more than one URL in the config were only able to get the toggles written file for the first URL, and then the rest were failing. This solution allows multiple URLs per service without removing memoization in the custom command. This works well (no failures) but if you have any ideas for improvement I'm open.